__test__/fixtures/graveyard.jsx

```diff
@@ -0,0 +1,40 @@
+import { format } from 'date-fns';
+
+var now = new Date();
```

Comment:
```suggestion
var now = new Date('2018-10-01');
```

Maybe test from a specific date instead of assumed `now`. This ensures snapshots with [relative date phrasing](https://github.com/codyogden/killedbygoogle/blob/ffa3f49d02af27ec983143bf638ceb29a0899cac/__test__/components/__snapshots__/Item.ThemeHalloween.test.jsx.snap#L26) like "10 months ago" will always remain the same.